File: AssetManager/GlobalObjects/GlobalInstances.vb
Module GlobalInstances

    Public Class SibiAttributes
        Public StatusType() As AttributeDataStruct
        Public ItemStatusType() As AttributeDataStruct
        Public RequestType() As AttributeDataStruct
        Public AttachFolder() As AttributeDataStruct
    End Class

    Public Class DeviceAttributes
        Public Locations() As AttributeDataStruct
        Public ChangeType() As AttributeDataStruct
        Public EquipType() As AttributeDataStruct
        Public OSType() As AttributeDataStruct
        Public StatusType() As AttributeDataStruct
    End Class

    Public DeviceAttribute As New DeviceAttributes
    Public SibiAttribute As New SibiAttributes
    Public MunisFunc As New MunisFunctions
    Public AssetFunc As New AssetManagerFunctions
    Public FTPFunc As New FtpFunctions

End Module

File: AssetManager/GlobalObjects/GlobalInstances.cs
namespace AssetManager
{
    internal static class GlobalInstances
    {
        public class SibiAttributes
        {
            public AttributeDataStruct[] StatusType;
            public AttributeDataStruct[] ItemStatusType;
            public AttributeDataStruct[] RequestType;
            public AttributeDataStruct[] AttachFolder;
        }

        public class DeviceAttributes
        {
            public AttributeDataStruct[] Locations;
            public AttributeDataStruct[] ChangeType;
            public AttributeDataStruct[] EquipType;
            public AttributeDataStruct[] OSType;
            public AttributeDataStruct[] StatusType;
        }

        public static DeviceAttributes DeviceAttribute = new DeviceAttributes();
        public static SibiAttributes SibiAttribute = new SibiAttributes();
        public static MunisFunctions MunisFunc = new MunisFunctions();
        public static AssetManagerFunctions AssetFunc = new AssetManagerFunctions();

        public static FtpFunctions FTPFunc = new FtpFunctions();
    }
}